pkg/locspec/locations.go

```diff
@@ -434,6 +438,30 @@ func (loc *NormalLocationSpec) Find(t *proc.Target, processArgs []string, scope
 	return []api.Location{addressesToLocation(addrs)}, nil
 }
 
+func substitutePath(path string, rules [][2]string) string {
```

Comment:
I'm not a big fan of duplicating this from the terminal package. What are your thoughts on moving this logic here instead and exporting it such that the frontend can use it as well?

Reply:
Done.